ipwhois/tests/test_asn.py

```diff
@@ -136,8 +136,30 @@ def test_parse_fields_http(self):
             self.fail('Unexpected exception raised: {0}'.format(e))
 
     def test_lookup(self):
-        # TODO: need to modify asn.json for this.
-        return NotImplemented
+        data_dir = path.dirname(__file__)
+
+        with io.open(str(data_dir) + '/asn.json', 'r') as \
+                data_file:
+            data = json.load(data_file)
+
+        for key, val in data.items():
+
+            log.debug('Testing: {0}'.format(key))
+            net = Net(key)
+            obj = IPASN(net)
+
+            try:
+
+                self.assertIsInstance(obj.lookup(), dict)
+                self.assertIsInstance(obj.lookup(asn_alts=['http']), dict)
```

Comment:
asn_alts is removed next release, and has already been removed from the dev branch.

Reply:
Please only remove line 154 (with the asn_alts argument).